app/Import/AudioImport.php

```diff
@@ -162,7 +161,6 @@ public function execute()
     // Keep track of which masters and transfers to update in Solr
     $masters = array();
     $transfers = array();
-    $items = array();
```

Comment:
Reference to $items on line 168 that can be removed now.

Reply:
removed!